doc/source/whatsnew/v2.3.0.rst

```diff
@@ -208,7 +208,7 @@ ExtensionArray
 
 Styler
 ^^^^^^
--
+- Enh in :meth:`Styler.set_tooltips` allowing users to store tooltips as title attribute on <td> elements. Benefits include multiline tooltip support and size footprint at losing the ability to style the tooltip text.
```

Comment:
To my understanding the `<span>` method pandas implements would not support multiline tooltips, you cannot word wrap in the span. So when the tooltip is larger or multiline, and it would spill out of the <td> then you lose text. 

I have not tested this and am not an expert on CSS/HTML by any means.

Reply:
I think there would probably be ways of controlling it actually, but I cannot remember having done it so am also not 100%. I would probably just say that this is an alternate, common way of also adding tooltips.

Reply:
This SO post makes it seems like it may be. Perhaps I'll investigate this, if its the case then it should alleviate your comments about that copied loop.

https://stackoverflow.com/a/28699424/9063378

Reply:
Although if I follow through the link and look at the example its not very pretty.

https://www.css3.info/tooltips-with-css3/
https://www.css3.info/wp-content/uploads/2008/02/tooltip.html